lib/core-integration/src/Test/Integration/Scenario/API/Shelley/StakePools.hs

```diff
@@ -879,7 +875,7 @@ spec = describe "SHELLEY_STAKE_POOLS" $ do
             verify rQuit
                 [ expectResponseCode HTTP.status202
                 , expectField (#status . #getApiT) (`shouldBe` Pending)
-                , expectField (#direction . #getApiT) (`shouldBe` Outgoing)
+                , expectField (#direction . #getApiT) (`shouldBe` Incoming)
```

Comment:
cc @piotr-iohk

Reply:
Isn't this weird?

Reply:
Or are we getting the deposit back now?

Reply:
We do want Incoming because of the deposit indeed, there's more coming in than out, and that's how we define Incoming